configs/sst_desktop_firmware_bootloaders-fwupd.yaml

```diff
@@ -7,6 +7,7 @@ data:
 
   packages:
   - fwupd
+  - fwupd-efi
```

Comment:
If this package is only intended for ELN/RHEL 10, then please copy this workload to a new file and make the change there. Then set the `labels:` appropriately.

If this package is being added also to c9s, then the current patch is fine.

Reply:
Let me investigate the C9S part and I'll get back to you and act accordingly.